commands/logs.js

```diff
@@ -0,0 +1,35 @@
+import fs from 'node:fs/promises'
+import { Tail } from 'tail'
+import { paths } from '../lib/paths.js'
+import { join } from 'node:path'
+
+const maybeCreateLogFile = async path => {
+  try {
+    await fs.writeFile(path, '', { flag: 'wx' })
+  } catch (err) {
+    if (err.code !== 'EEXIST') {
+      throw err
+    }
+  }
+}
+
+const followLogs = path => {
+  const tail = new Tail(path, { nLines: 10 })
+  tail.on('line', line => console.log(line))
+}
+
+const getLogs = async path => {
+  process.stdout.end(await fs.readFile(path))
```

Comment:
I find it surprising that `getLogs` has a significant side-effect of closing the `stdout` stream.

I am proposing two changes:
1. Rename `getLogs` to something like `printLogs`
2. Can we write the logs without closing stdout?

Reply:
Good catch! I was over-eager with the `.end`